newIDE/app/src/EventsSheet/EventsTree/Instruction.js

```diff
@@ -187,6 +224,15 @@ const Instruction = (props: Props) => {
             parameterIndex >= 0 && parameterIndex < parametersCount;
 
           if (!isParameter) {
+            if (value === 'Unknown or unsupported instruction') {
```

Comment:
this feels like a magic string, should we have a Constant somewhere we can import and re-use here? like `UNKNOWN_INSTRUCTION_VALUE`

Reply:
It's defined in the C++ part so I'm not sure it's easily doable

Reply:
oh right, then probably not.